templates/ctl-scaffold/flake.nix

```diff
@@ -24,14 +25,61 @@
         "aarch64-darwin"
       ];
       perSystem = nixpkgs.lib.genAttrs supportedSystems;
+
+      # generate `pkgs` with CTL's overlays applied. This gives you access to
+      # various additional packages. The versions are the same as those that CTL uses.
       nixpkgsFor = system: import nixpkgs {
         inherit system;
         overlays = [
           ctl.overlays.purescript
+          # This one is optional. If you set `ctl-server.enable = true;`
+          # in the runtime config as in our nix/runtime.nix, you must enable this
+          # overlay. `ctl-server` itself is **only** required when using
+          # CTL's `applyArgs` function
           ctl.overlays.ctl-server
           ctl.overlays.runtime
         ];
       };
+
+
+      # The configuration for the CTL runtime, which will be passed to the
+      # expression that builds the JSON file used by Arion. This value can be
+      # shared between `buildCtlRuntime` and `launchCtlRuntime`, as shown below
+      #
+      # You can refer to the final configuration value by passing a function
+      # that takes a single arugment. Alternatively, you can pass an attrset
+      # directly
+      runtimeConfig = { };
```

Comment:
Let's add a link to `nix/runtime.nix` `defautlConfig` here, so that the users will be able to know where to copy-paste from.